base/logging_new.go

```diff
@@ -0,0 +1,137 @@
+package base
+
+import (
+	"log"
+	"os"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	// RFC3339 + 3 digit milli
+	timestampFormat = "2006-01-02T15:04:05.000Z07:00"
+)
+
+var (
+	ErrInvalidLogFilePath   = errors.New("invalid LogFilePath")
+	ErrInvalidLoggingMaxAge = errors.New("invalid MaxAge")
+
+	// We'll initilise a default consoleLogger so we can still log stuff before/during parsing logging configs.
```

Comment:
If there's a benefit to doing this (as opposed to just using simple log.Printf style logging) prior to config logging, it would be useful to add that to the comment.  (maybe timestamp info formatting?)